app/middleware/setLocals.js

```diff
@@ -1,22 +1,27 @@
 const notFound = require('../middleware/renderer').notFound;
 
+function hostNameAndProtocol(req) {
+  return `https://${req.hostname}`;
```

Comment:
Should the protocol be determined from the request, rather than assuming it is always https?